app/src/main/java/nz/org/cacophony/birdmonitor/BootReceiver.java

```diff
@@ -51,19 +51,11 @@ public void run() {
                     long timeout = 1000 * 60 * 2;  // give the boot stuff two minutes to run - but the enable flight mode does not seem to be working (however long I wait).
                     wakeLock.acquire(timeout); // finally never seems to run which is why I used a timeout on the wakelock creation
                     try {
-
-
-                        ToneGenerator toneGen1 = new ToneGenerator(AudioManager.STREAM_MUSIC, 100);
-                        toneGen1.startTone(ToneGenerator.TONE_CDMA_CALL_SIGNAL_ISDN_NORMAL, 1000);
```

Comment:
Did we find out why this was originally added?

Reply:
nope i decided it was just left in there by mistake